src/box/replication.cc

```diff
@@ -1160,6 +1197,8 @@ replicaset_connect(const struct uri_set *uris,
 	/* Now all the appliers are connected, update the replica set. */
 	replicaset_update(appliers, count, keep_connect);
 	appliers_guard.is_active = false;
+	replication_fiber = NULL;
```

Comment:
Bad name: it's not a replication fiber; I think it's better call it after the function, i.e. `replicaset_connect_fiber`.

Reply:
Fixed.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/replication.cc b/src/box/replication.cc
index 55120ffd3..73b6fe561 100644
--- a/src/box/replication.cc
+++ b/src/box/replication.cc
@@ -61,8 +61,13 @@ double replication_synchro_timeout = 5.0; /* seconds */
 double replication_sync_timeout = 300.0; /* seconds */
 bool replication_skip_conflict = false;
 int replication_threads = 1;
+/**
+ * Fiber executing replicaset_connect. NULL if the function
+ * is not being executed.
+ */
+struct fiber *replication_connect_fiber;
+
+/** Condition that replicaset_connect finished execution. */
 struct fiber_cond replication_connect_cond;
-struct fiber *replication_fiber;
+
 /** If set then replication shutdown is started. */
 static bool replication_is_shutting_down;
 
@@ -235,9 +240,9 @@ void
 replication_shutdown(void)
 {
 	replication_is_shutting_down = true;
-	if (replication_fiber != NULL)
-		fiber_cancel(replication_fiber);
-	while (replication_fiber != NULL)
+	if (replication_connect_fiber != NULL)
+		fiber_cancel(replication_connect_fiber);
+	while (replication_connect_fiber != NULL)
 		fiber_cond_wait(&replication_connect_cond);
 
 	struct replica *replica;
@@ -1079,8 +1084,8 @@ replicaset_connect(const struct uri_set *uris,
 		tnt_raise(ClientError, ER_CFG, "replication",
 			  "too many replicas");
 	}
-	assert(replication_fiber == NULL);
-	replication_fiber = fiber();
+	assert(replication_connect_fiber == NULL);
+	replication_connect_fiber = fiber();
 	int count = 0;
 	struct applier *appliers[VCLOCK_MAX] = {};
 	auto appliers_guard = make_scoped_guard([&]{
@@ -1089,7 +1094,7 @@ replicaset_connect(const struct uri_set *uris,
 			applier_stop(applier);
 			applier_delete(applier);
 		}
-		replication_fiber = NULL;
+		replication_connect_fiber = NULL;
 		fiber_cond_signal(&replication_connect_cond);
 	});
 	for (; count < uris->uri_count; count++)
@@ -1196,7 +1201,7 @@ replicaset_connect(const struct uri_set *uris,
 	/* Now all the appliers are connected, update the replica set. */
 	replicaset_update(appliers, count, keep_connect);
 	appliers_guard.is_active = false;
-	replication_fiber = NULL;
+	replication_connect_fiber = NULL;
 	fiber_cond_signal(&replication_connect_cond);
 	uri_set_destroy(&replication_uris);
 	uri_set_copy(&replication_uris, uris);
```

</details>